tests/rptest/services/redpanda_installer.py

```diff
@@ -23,6 +24,9 @@
 # E.g. "v22.1.1-rc1-1373-g77f868..."
 VERSION_RE = re.compile(".*v(\\d+)\\.(\\d+)\\.(\\d+).*")
 
+RELEASES_CACHE_FILE = "/tmp/redpanda_releases.json"
```

Comment:
I believe this needs to be under the redpanda_installs directory for it to be shared across mount points, unless we also share /tmp across containers (I don't think we do, looking at https://github.com/redpanda-data/vtools/blob/2a7be703886f80712088340c6c69ae72a270cb0c/taskfiles/rp.yml#L312-L335)

Reply:
This isn't meant to be shared across containers, just used from the `ducktape` container

Reply:
Ah I see. I was initially worried about having the file not be exposed to users in their filesystems, since the tmp file lives in the containers' /tmp directory, but it shouldn't be problematic given the TTL.